public/styles.css

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Style suggestion: Leave a blank line between CSS blocks. (I think that's the right way... not sure)